js/packages/components/chat/MultiMember.tsx

```diff
@@ -73,7 +73,7 @@ const HeaderMultiMember: React.FC<{
 			>
 				<TouchableOpacity
 					style={[flex.tiny, flex.justify.center, flex.align.center]}
-					onPress={goBack}
+					onPress={() => navigate.main.home()}
```

Comment:
are you sure these changes are wanted ?

Reply:
Hi @n0izn0iz , I have updated this code, as we need to navigate to home screen from Group Chat when new group is created it redirect to Group Chat. If goBack is implemented then it will go back to Group Creation screen.

Reply:
The right fix ix to use  `reset` instead of `navigate` in Group Creation then

Reply:
I believe this is addressed in #1716

Reply:
I believe the fix is already merged in https://github.com/berty/berty/pull/2916